hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -18,12 +19,824 @@
+            - Verify that the device ACK the setup packet and prepare for the data stage.
+            '''
+      stage: V1
+      tests: ["usbdev_setup_stage"]
+    }
+    {
+      name: in_data_stage
+      desc: '''
+            Verify the IN Transfer with Device Descriptor Data.
+
+            - Issue an IN token to request control data from the device.
+            - Verify that the device should respond with a DATA packet containing the information
+              required by host or a NAK packet if there is no data available.
+            '''
+      stage: V1
+      tests: ["usbdev_in_data_stage"]
+    }
+    {
+      name: out_data_stage
+      desc: '''
+            Verify the OUT Transfer with DEVICE Settings.
+
+            - Send an OUT token followed by a data packet containing control data to the device.
+            - Verify that the device should acknowledge the successful receipt of data by sending
+              an ACK packet or return a NAK if it is still processing previous data.
+            '''
+      stage: V1
+      tests: ["usbdev_out_data_stage"]
+    }
+    {
+      name: out_status_stage
+      desc: '''
+            Verify the OUT Transfer for Status Reporting.
+
+            - If the host used IN tokens during data stage then status stage is initiated with an OUT token.
+            - Send an OUT token followed by a zero-length DATA0 data packet to the device.
+            - Verify that the device should acknowledge the successful receipt of the status request by
+              sending an ACK packet or return NAK if it is still processing or STALL if an error occurred.
+            '''
+      stage: V1
+      tests: ["usbdev_out_status_stage"]
+    }
+    {
+      name: in_status_stage
+      desc: '''
+            Verify the IN Transfer for Status Reporting.
+
+            - If the host used OUT tokens during data stage then status stage must start with an IN Token.
+            - Verify that the device must respond with zero length Data0 packet for succesful completion
+              STALL for error and NAK for busy followed by ACK from Host.
+            '''
+      stage: V1
+      tests: ["usbdev_in_status_stage"]
+    }
+    {
+      name: endpoint_access
+      desc: '''
+            Verify the accessibility of all endpoint when endpoints are enabled.
+
+            - Access all the endpoints of the device in IN and OUT direction by enabling ep_out_enable and
+              ep_in_enable along with rxenable_out for OUT transaction.
+            - Verify that the device should allow access to all endpoints by responding with ACK for OUT
+              accesses and data_packet for IN Accesses.
+            '''
+      stage: V1
+      tests: ["usbdev_endpoint_access"]
+    }
+    {
+      name: disable_endpoint
+      desc: '''
+            Verify the ignorance of transmission/reception of packets when endpoints are disabled.
```

Comment:
Suggestion: Verify that packet transmissions (IN and SETUP) and reception requests (IN) are ignored when endpoints are disabled.